packages/web/src/components/gcds-error-summary/stories/gcds-error-summary.stories.tsx

```diff
@@ -36,15 +37,15 @@ const Template = args =>
   `
 <!-- Web component code (Angular, Vue) -->
 <gcds-error-summary
-  ${args.listen && !args.errorLinks ? `listen` : null}
+  ${args.listen && args.errorLinks ? `listen` : null}
```

Comment:
This logic is to set either `listen` or `error-links` will need to be set back to `${args.listen && !args.errorLinks ? `listen` : null}`